.github/workflows/tests_and_release.yml

```diff
@@ -0,0 +1,65 @@
+name: Validate and Release connector
+on:
+  pull_request:
+  push:
```

Comment:
This will probably trigger the build twice, I don't know exactly. `push` events should be limited to run on `main`, afaik.